eng/pipelines/coreclr/templates/crossdac-build.yml

```
@@ -45,11 +45,18 @@ steps:
           $availableFiles | %{ cp $_.FullName $targetDir -v }
         }
 
-        CopyAndVerifyCrossOsAssets -CrossDacDir $(buildMuslDacRootFolderPath) -TargetDir '$(buildMuslDacStagingPath)/${{ parameters.hostArchType }}'
-        CopyAndVerifyCrossOsAssets -CrossDacDir $(buildLinuxDacRootFolderPath) -TargetDir '$(buildLinuxDacStagingPath)/${{ parameters.hostArchType }}'
+        $buildMuslDacRootFolderPath = "$(Build.SourcesDirectory)/artifacts/bin/coreclr/alpine.${{ parameters.archType }}.$(buildConfigUpper)"
+        $buildMuslStagingPath = "$(crossDacArtifactPath)/Linux_musl.${{ parameters.archType }}.$(buildConfigUpper)/${{ parameters.hostArchType }}"
+        
+        $buildLinuxDacRootFolderPath = "$(Build.SourcesDirectory)/artifacts/bin/coreclr/Linux.${{ parameters.archType }}.$(buildConfigUpper)"
+        $buildLinuxDacStagingPath = "$(crossDacArtifactPath)/Linux.${{ parameters.archType }}.$(buildConfigUpper)/${{ parameters.hostArchType }}"
+        
+        
+        CopyAndVerifyCrossOsAssets -CrossDacDir '$buildMuslDacRootFolderPath' -TargetDir '$buildMuslStagingPath'
+        CopyAndVerifyCrossOsAssets -CrossDacDir '$buildLinuxDacRootFolderPath' -TargetDir '$buildLinuxDacStagingPath'
```

```suggestion
        CopyAndVerifyCrossOsAssets -CrossDacDir $buildMuslDacRootFolderPath -TargetDir $buildMuslStagingPath
        CopyAndVerifyCrossOsAssets -CrossDacDir $buildLinuxDacRootFolderPath -TargetDir $buildLinuxDacStagingPath
```
I do not think that this can be quoted. The variable name is treated as literal string when quoted.

---

They were meant to be single quotes, for spaces in path. Sorry for that.